addon/components/rdfa-editor-container.hbs

```diff
@@ -0,0 +1,45 @@
+{{#if @busy}}
+  <AuBodyContainer>
+    <div class="au-c-rdfa-editor">
+      <div class="say-container say-container--sidebar-left say-container--paper say-container--sidebar-right">
+        <div class="say-container__main">
+          <div class="say-editor rdfa-annotations rdfa-annotations-highlight rdfa-annotations-hover">
+            <div class="say-editor__paper">
+              <div class="au-c-scanner">
+                <div class="au-c-scanner__text">
+                  <AuLoader @padding="small"/>
+                  <AuHelpText @size="large">{{if @busyText @busyText
+                                                 (t "rdfaEditorContainer.defaultBusyText") }}
+                  </AuHelpText>
+                </div>
+                <span class="au-c-scanner__bar"></span>
+              </div>
+              <div class="say-editor__inner say-content">
+                {{html-safe this.editor.htmlContent}}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </AuBodyContainer>
+{{else}}
+  <AuBodyContainer 
+    vocab="{{this.vocab}}" 
+    {{did-insert this.setPrefix}}
```

Comment:
The addon doesn't list `@ember/render-modifiers` as a dependency so at the moment this only works by accident since another addon brings it in. 

Are these rdfa attributes needed for the editor to work? I don't see it documented in the editor readme so if it's not needed I would just delete the wrapper altogether?

Maybe @cecemel has a different opinion.

Reply:
Depends on what we want to do respect to rdfa with the rich text editor, the point of this prefixes + vocab + everything is to provide a wrapper around the content, for example in GN the wrapper is the behandelingVanAgendapunt, and all the editor content is inside that.
This wrapper sets the context and the prefixes

Reply:
The way I understood it is that rdfa-editor is just an implementation detail for the rich-text field. Users should just be able to edit it in a user-friendly way. So I would make it highly opinionated for that use-case and not think about the rdfa editing for now.

So I would just disable everything that we don't need. Things like the sidebar, "paper", prefix, etc. We can always refactor things later when we actually need rdfa editing and there is obvious logic duplication going on.

It would make adding styles for this a lot easier as well. We could probably just borrow the setup that's used for the AuTextarea component but with a toolbar at the top. I feel we can even manage a first iteration without any designer support 😄.

Reply:
I don't think for the rich-text-editor this should be needed. In theory we should be able to swap it with any other rich-text-editor